python-data-analysis/python-data-analysis-core/analysis-environments/what-are-analysis-environments.md

```diff
@@ -0,0 +1,45 @@
+---
+author: Stefan-Stojanovic
+
+type: normal
+
+category: must-know
+
+---
+
+# What Are Analysis Environments?
+
+---
+## Content
+
+When it comes to coding in Python, there are several environments you can choose from.
+
+Analysis environments are shells, [IDE's](https://www.enki.com/glossary/general/ide), and other ways where you can test, run, modify, execute, and so on..., a programming language's code.
+
+For instance, the most basic environment you can write Python is in your OS's terminal[1]. 
+
+You can write and test code here by first initializing the [interpreter](https://www.enki.com/glossary/general/interpreter) by typing `python` in your terminal.
+
+> 💡 Different versions of Python will have different keywords to launch the basic interpreter. However, the most common ones are `py` and `python`.
+
+Even though most operating systems come with a pre-installed version of the Python interpreter it is not the recommended way for coding. This is because there are far superior environments with better interpreters for coding in Python.
+
+Some of these are:
+- Jupyter Lab
+- IDLE
+- nteract
+- PyScripter
+- Google colab
+- VScode
+- and many more
+
+> 💡 Some of the more superior features are type suggestions, syntax highlighting, in-line graphics, and so on."
+
+In this course, we will be using the Jupyter Lab environment.
+
+---
+## Footnotes
+
+[1: Terminal Gif]
+
+[terminal-gif](https://img.enkipro.com/0bf78b2222dca114a879cc242715adc3.gif)
```

Comment:
missing exclamation mark to turn it into an image (instead of a link)
```suggestion
![terminal-gif](https://img.enkipro.com/0bf78b2222dca114a879cc242715adc3.gif)
```